src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -22,6 +22,87 @@ private enum EscapeLevel : uint
             ReturnOnly = ReturnOnlyScope,
         }
 
+        /// <summary>
+        /// Encapsulates a property or indexer and the accessor(s) on it that were used. The particular
+        /// indexer used is important as it impacts ref safety analysis
+        /// </summary>
+        private readonly struct MethodInfo
+        {
+            internal Symbol Symbol { get; }
+            internal MethodSymbol? Method { get; }
```

Comment:
Is `Method` ever `null`? If not, consider removing `?` annotation.

Reply:
Unfortunately it very much can be `null` in error cases. For example when an indexer is used as an rvalue but there is only a `set` you can get cases where `Method` is `null`.

Reply:
Added some doc comments to clarify